pyenvisalink/envisalink_base_client.py

```diff
@@ -195,22 +172,22 @@ async def disconnect(self):
 
         self._writer = None
         self._reader = None
-            
-    async def send_data(self, data, logData = None):
+
+    async def send_data(self, data, logData=None):
         """Raw data send- just make sure it's encoded properly and logged."""
         # Scrub the password and alarm code if necessary
         if not logData:
             logData = self.scrub_sensitive_data(data)
-        _LOGGER.debug('TX > %s', logData.encode('ascii'))
+        _LOGGER.debug("TX > %s", logData.encode("ascii"))
 
         try:
-            self._writer.write((data + '\r\n').encode('ascii'))
+            self._writer.write((data + "\r\n").encode("ascii"))
```

Comment:
Should probably use `self._terminator` here that you introduced for the `readuntil()` call.

Reply:
Will do.